src/quick-lint-js/fe/diagnostic-types.h

```diff
@@ -211,6 +211,21 @@
               "'async static' is not allowed; write 'static async' instead"),   \
           async_static))                                                        \
                                                                                 \
+  QLJS_DIAG_TYPE(                                                               \
+      diag_async_export_method, "E0326", diagnostic_severity::error,            \
+      { source_code_span async_export; },                                       \
+      MESSAGE(                                                                  \
+          QLJS_TRANSLATABLE(                                                    \
+              "'async export' is not allowed; write 'export async' instead"),   \
+          async_export))                                                        \
+                                                                                \
+  QLJS_DIAG_TYPE(                                                               \
+      diag_function_async_method, "E0327", diagnostic_severity::error,          \
```

Comment:
This should be called `diag_function_async_function`, because it's for a function, not a class method.